skfem/importers/meshio.py

```diff
@@ -1,6 +1,7 @@
 """Import any formats supported by meshio."""
 
 from typing import Tuple
+from collections import OrderedDict
```

Comment:
I had to check whether `OrderedDict` was still required or whether a standard [`dict`](https://docs.python.org/dev/library/stdtypes.html#dict) would suffice.  It is till support for 3.6 is abandoned.

> Changed in version 3.7: Dictionary order is guaranteed to be insertion order.

Reply:
Ok, let's have it there for now then. I think 3.6 is supported officially until the beginning of 22?